problem_builder/mentoring.py

```diff
@@ -817,13 +858,9 @@ def author_edit_view(self, context):
         """
         Add some HTML to the author view that allows authors to add child blocks.
         """
-        fragment = Fragment(u'<div class="mentoring">')  # This DIV is needed for CSS to apply to the previews
-        self.render_children(context, fragment, can_reorder=True, can_add=False)
-        fragment.add_content(u'</div>')
-        fragment.add_content(loader.render_template('templates/html/mentoring_add_buttons.html', {}))
```

Comment:
Now that were' using `StudioContainerWithNestedXBlocksMixin`, do we still need the code in this file: https://github.com/open-craft/problem-builder/blob/4ffa21c36b534f9b86f50b128815800a33b69899/problem_builder/public/js/mentoring_edit.js or is that handled by the mixin now? It might be worth checking and remove that file if it's not needed anymore.